promql/comparer/comparer.go

```diff
@@ -169,3 +170,23 @@ func addDropResultLabelsOptions(queryTweaks []*config.QueryTweak, options *cmp.O
 		}
 	}
 }
+
+func addCaseInsensitiveCompareOptions(queryTweaks []*config.QueryTweak, options *cmp.Options) {
+	for _, rt := range queryTweaks {
+		if rt.IgnoreCase {
+			*options = append(
+				*options,
+				// Translate sample values into lowercase.
+				cmp.Transformer("TranslateToLowerCase",
+					func(in model.Metric) model.Metric {
+						m := make(map[model.LabelName]model.LabelValue)
```

Comment:
Style nit: only use `make()` when it is required/has an advantage. Otherwise let's go for the simpler literal initialization:

```suggestion
						m := map[model.LabelName]model.LabelValue{}
```